integrated_channels/moodle/client.py

```diff
@@ -5,6 +5,7 @@
 
 import json
 import requests
+from six.moves.urllib.parse import urljoin  # pylint: disable=import-error
```

Comment:
hmmm why this specific import path instead of just `from urllib.parse`? Especially curious since there is an import from urllib.parse already in this file (I imported urlencode because I wanted the base functionality not Django's wrapped version)

Reply:
Bad merge and shamelessly stolen from elsewhere, I'll update and simplify